app/models/user.rb

```diff
@@ -1,10 +1,17 @@
 class User < ApplicationRecord
+  attr_accessor :email_confirmation
+
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly enable `:confirmable` for devise

Reply:
Done!.
- Email sent to confirmate the new account
![image](https://user-images.githubusercontent.com/53025062/224111129-2c1191cc-e80a-457f-9966-794636045f3e.png)

- New account confirmed:
![image](https://user-images.githubusercontent.com/53025062/224111358-e41e1f66-1fc0-44f9-bdd3-1efd2bb52ed3.png)

- Activate and style for "Resend confirmation instructions"
![image](https://user-images.githubusercontent.com/53025062/224112699-06992c00-40e6-4603-8b4e-1710ad6618c1.png)

![image](https://user-images.githubusercontent.com/53025062/224112856-75b23e55-4b3c-4996-9c34-d981df553688.png)

![image](https://user-images.githubusercontent.com/53025062/224113164-36487014-b4d6-4cbb-815c-ded70fab5e39.png)